src/data/filterConstants.js

```diff
@@ -405,11 +405,11 @@ export const regions = [
     continent: continentEnum.europe,
     geopolitical: geopoliticalEnum.europe,
   },
-  {
```

Comment:
From the backend perspective, removing all the non-viable regions will get up closer to more manageable query strings.